### ISSUE TITLE
Un-hardcoding danish-language labels

### ISSUE THREAD

**Issue by chcederquist** [chcederquist/react-dkfds#3]
All currently hardcoded labels and aria-labels should be configurable through props

Current components(non-exhaustive) with hardcoded labels:
- Accordion (open/close all)
- Alert (icon aria label)
- Back To Top
